scripts/check.sh

```diff
@@ -167,18 +167,16 @@ fi
 # first commit on a non-master branch, TRAVIS_COMMIT_RANGE is not set and
 # START_REVISION is "master" instead of a range.
 
-if [[ -z "${TRAVIS_COMMIT_RANGE}" ]]; then
-  # Figure out if we have access to master. If not add it to the repo.
+# If needed, check if we have access to master and add it to the repo.
```

Comment:
No really, I think all we need is a one-line change. Change the line near the top that sets the default:

```
START_REVISION="master"
```
Becomes
```
START_REVISION="origin/master"
```

(Actually there are a few other references to `master` in comments should also be changed, so it's not quite a one line change in the end.)

Reply:
@wilhuff That doesn't work because there is no TRAVIS_COMMIT_RANGE. See https://travis-ci.org/firebase/firebase-ios-sdk/builds/533544992 and https://github.com/firebase/firebase-ios-sdk/commit/7a7bd2f83a1a9d6e31e39b950b8ba864c54e75bd

Reply:
With set -x - https://travis-ci.org/firebase/firebase-ios-sdk/builds/533549668

Reply:
Ah, I see now.

OK. The current state of this PR does not include the change to the default for `START_REVISION` (that was in your `set -x` output), and it seems like it should. The `git remote set-branches` fix we apply only guarantees that `origin/master` will exist but doesn't create a `master` tracking branch.

Otherwise LGTM.

Reply:
Thanks. Done.